tests/test_scripts.py

```diff
@@ -3218,3 +3218,17 @@ def test_roundtrip_commented_format():
     r = open(infile).read()
     r2 = open(outfile).read()
     assert r == r2, (r, r2)
+
+
+def test_multirename_fasta():
+
+    infile1 = utils.get_temp_filename('test-multi.fa')
+    multioutfile = utils.get_temp_filename('out.fa')
+    infile2 = utils.get_temp_filename('out.fa')
+    shutil.copyfile(utils.get_test_data('test-multi.fa'), infile1)
+    shutil.copyfile(utils.get_test_data('multi-output.fa'), infile2)
+    args = ['assembly', infile1]
+    _, out, err = utils.runscript('multi-rename.py', args)
+    r = open(infile2).read()
+    for k in r:
+        assert k in out, (k, out)
```

Comment:
this assertion is checking that every character in 'r' in present somewhere in 'out', which is probably not what you want?

if you are trying to compare multi-output.fa to the output of multi-rename, you probably want to do:

`assert r == out`

or some such.